app/models/form.rb

```diff
@@ -6,8 +6,8 @@ class Form < ApplicationRecord
   has_many :questions, through: :form_questions
   has_many :response_sets, through: :form_questions
 
-  has_one :surveillance_system
-  has_one :surveillance_program
+  belongs_to :surveillance_system
+  belongs_to :surveillance_program
```

Comment:
@kierk this currently has a form being associated with a single system or program.  The one to many in this case is a system or program can have many forms. This association will probably move to the survey model in the near future but would remain as a survey belongs to just one system or program.  Long winded answer for nothing to see here.